### ISSUE TITLE
relecture treguy 7.3

### ISSUE THREAD

**Issue by FabienTregan** [uggla/rfs_07#2]
https://github.com/uggla/rfs_07/blob/174fb81c6cc844b35ad0bba19ac6f95d76053326/solution_03/src/main.rs#L20-L26

comme on est *vraiment* des grosses feignasses, on va probablement faire :
```rust
 use crate::Planets::*;
[...]
 let destinations = [ 
     Tatooine, 
     Alderaan, 
     Coruscant, 
     Dagobah, 
     Mustafar, 
 ]; 
```

**Comment by uggla** [comment]
Possible, mais je trouve que l'on perd beaucoup en lisibilité. Donc je vais laisser le code comme il est dans l'exemple.
Personnellement, je fais pas ça car après j'ai du mal à savoir d'où vient cet enum. :)

**Comment by FabienTregan** [comment]
Oui, avant je n'aimais pas pour la meme raison que toi (pas lié a rust). Mais depuis que les IDE affichent les types (ici rust analyzer afficherait un truc du genre `let destination :[Planets, 5]`, j'ai pris cette habitude.

Pour les switchs en particulier je trouve que ca allège bien. D'ailleurs la nouvelle notation pour les switchs en java t'oblige à le faire quand tu switch sur une énum.
